src/naga_http/server.clj

```diff
@@ -4,15 +4,68 @@
   (:require [cheshire.core :as json]
             [compojure.core :refer :all]
             [compojure.route :as route]
-            naga.cli
-            [ring.middleware.defaults :refer [wrap-defaults api-defaults]]))
+            [naga.lang.pabu :as pabu]
+            [naga.engine :as e]
+            [naga.rules :as r]
+            [naga.store :as store]
+            [naga.data :as data]
+            [ring.middleware.defaults :refer [wrap-defaults api-defaults]])
+  (:import [java.util Date]))
+
+(def plain-headers {"Content-Type" "text/plain"})
+
+(def json-headers {"Content-Type" "application/json"})
+
+(def programs (atom {}))
+
+(defn uuid-str []
+  (-> (java.util.UUID/randomUUID)
```

Comment:
This will fail the kibit check, since it asks that threading go through more than one step.